### PR TITLE
workers.openstack: Add further documentation about nova_args

### PR DESCRIPTION
**Pull request by cmouse**

### CODE REVIEW

File: master/docs/manual/cfg-workers-openstack.rst [hunk 1]
```diff
@@ -152,6 +152,28 @@ When the instance is destroyed, the volume will be terminated as well.
     ]
 
 
+The ``nova_args`` can be used to specify additional arguments for the novaclient. For example network mappings,
```
Comment (tardyp) [inline master/docs/manual/cfg-workers-openstack.rst]: we have the policy of maintaining one line per sentence in the doc (for new doc, and we try to slowly transition for old doc)

File: master/docs/manual/cfg-workers-openstack.rst [hunk 2]
```diff
@@ -152,6 +152,28 @@ When the instance is destroyed, the volume will be terminated as well.
...
+which is required if your openstack tenancy has more than one network, and default cannot
+be determined. Please refer to your openstack manual whether it wants net-id or net-name.
+
+Other useful parameters are availability_zone, security_groups and config_drive. Refer to
+`Python bindings to the OpenStack Nova API <http://docs.openstack.org/developer/python-novaclient/>`
```
Comment (tardyp) [inline master/docs/manual/cfg-workers-openstack.rst]: I think rst requires a _ in the end: 

   ...openstack.org/developer/python-novaclient/>`_

**Comment by tardyp** [comment]
@sa2ajj might want to add something, as he is the master of the sphinx

**Comment by tardyp** [comment]
Please have a look at travis, there is a spelling issue (need to add an exception for openstack)

**Comment by cmouse** [comment]
Yeah, I'll take a look. I can't make heads or tails from appveyor/pr but the travis-ci/pr fail is clearer.

**Comment by tardyp** [comment]
Hi @cmouse could you make the few spell mistake fixes?

**Comment by cmouse** [comment]
Bah. Yes sorry. Slipped my mind.

File: master/docs/manual/cfg-workers-openstack.rst [hunk 3]
```diff
@@ -162,6 +162,28 @@ When the instance is destroyed, the volume will be terminated as well.
     ]
 
 
+The ``nova_args`` can be used to specify additional arguments for the novaclient.
+For example network mappings, which is required if your OpenStack tenancy has more than one network, and default cannot be determined.
+Please refer to your OpenStack manual whether it wants net-id or net-name.
+
+Other useful parameters are availability_zone, security_groups and config_drive.
```
Comment (rutsky) [inline master/docs/manual/cfg-workers-openstack.rst]: I would format Python parameters as raw block:

```
Other useful parameters are ``availability_zone``, ``security_groups`` and ``config_drive``.
```

Honestly I would expect Sphinx to report them as spelling errors...

File: master/docs/manual/cfg-workers-openstack.rst [hunk 4]
```diff
@@ -162,6 +162,28 @@ When the instance is destroyed, the volume will be terminated as well.
...
+        worker.OpenStackLatentWorker('bot2', 'sekrit',
+                    flavor=1, image='8ac9d4a4-5e03-48b0-acde-77a0345a9ab1',
+                    os_username='user', os_password='password',
+                    os_tenant_name='tenant',
+                    os_auth_url='http://127.0.0.1:35357/v2.0',
+                    nova_args={
+                      'nics': [
+                                {'net-id':'uid-of-network'}
+                              ]}
+
```
Comment (rutsky) [inline master/docs/manual/cfg-workers-openstack.rst]: Please add closing brackets to make this code syntatically-correct Python code (then Sphinx will highlight it).

**Comment by rutsky** [comment]
Currently built cods from this PR: https://circle-artifacts.com/gh/buildbot/buildbot/755/artifacts/0/home/ubuntu/buildbot/master/docs/_build/html/manual/cfg-workers-openstack.html

**Comment by rutsky** [comment]
@tardyp I remove `stalled` label, since @cmouse responded and added fix.

**Comment by cmouse** [comment]
Is this OK now?

**Comment by rutsky** [comment]
@cmouse, not yet. 
Notice that your code example is not formatted - it still not syntactically correct (https://circle-artifacts.com/gh/buildbot/buildbot/758/artifacts/0/home/ubuntu/buildbot/master/docs/_build/html/manual/cfg-workers-openstack.html), it misses closing `)` bracket:

```python
from buildbot.plugins import worker
c['workers'] = [
    worker.OpenStackLatentWorker('bot2', 'sekrit',
                flavor=1, image='8ac9d4a4-5e03-48b0-acde-77a0345a9ab1',
                os_username='user', os_password='password',
                os_tenant_name='tenant',
                os_auth_url='http://127.0.0.1:35357/v2.0',
                nova_args={
                  'nics': [
                            {'net-id':'uid-of-network'}
                          ]}) # <-- here
]
```

**Comment by rutsky** [comment]
Changes LGTM: https://circle-artifacts.com/gh/buildbot/buildbot/772/artifacts/0/home/ubuntu/buildbot/master/docs/_build/html/manual/cfg-workers-openstack.html

**Pull request merged**